tests/data/dummy_aea/aea-config.yaml

```diff
@@ -1,6 +1,6 @@
 aea_version: 0.1.7
 agent_name: Agent0
-authors: Fetch.AI Limited
```

Comment:
missing fingerprint field

Reply:
I don't think that we want fingerprint here.